source/server/options_impl.cc

```diff
@@ -220,6 +220,17 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv,
   }
 }
 
+void OptionsImpl::validate() {
+  // Exactly one of config_path and config_yaml should be specified.
```

Comment:
I checked. There is no test currently that tests it; I'll look at adding one. But it does look like the code that parses the configs accepts both.

So I decided to remove the new interface I had added and just do the checking for no configs in server.cc where the configs are parsed.

Reply:
There *is* a unit test for this in config_test.cc: testMerge(). This was bypassed by my previous code change since it relied on an explicit validate() call which did not occur in that test.